tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -67,33 +71,38 @@ class BasicHealthTest
 
     it should "fire a trigger when a message is posted to message hub" in withAssetCleaner(wskprops) {
         val currentTime = s"${System.currentTimeMillis}"
+        val triggerName = "/_/BasicHealthTestTrigger"
 
         (wp, assetHelper) =>
-            val triggerName = s"/_/dummyMessageHubTrigger-$currentTime"
-            println(s"Creating trigger ${triggerName}")
-
-            val feedCreationResult = assetHelper.withCleaner(wsk.trigger, triggerName) {
-                (trigger, _) =>
-                    trigger.create(triggerName, feed = Some(s"$messagingPackage/$messageHubFeed"), parameters = Map(
-                        "user" -> kafkaUtils.getAsJson("user"),
-                        "password" -> kafkaUtils.getAsJson("password"),
-                        "api_key" -> kafkaUtils.getAsJson("api_key"),
-                        "kafka_admin_url" -> kafkaUtils.getAsJson("kafka_admin_url"),
-                        "kafka_brokers_sasl" -> kafkaUtils.getAsJson("brokers"),
-                        "topic" -> topic.toJson))
-            }
+            val result = wsk.trigger.get(triggerName, expectedExitCode = DONTCARE_EXIT)
 
-            withActivation(wsk.activation, feedCreationResult, initialWait = 5 seconds, totalWait = 60 seconds) {
-                activation =>
-                    // should be successful
-                    activation.response.success shouldBe true
-            }
+            if(result.exitCode != 0) {
```

Comment:
Non-zero exit code doesn't guarantee the trigger doesn't exist. An exit code of 148 does.